src/style.css

```
@@ -0,0 +1,37 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+.search-input {
+  position: relative;
+  font-family: "Avenir", serif;
+  font-weight: 300;
+  color: #3d4e61;
+  width: 100%;
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  padding-left: 62px;
+  background-position: top 25px left 26px;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+}
+
+.mini-search-input {
```

```suggestion
.search-input--mini {
```
in BEM classes named like this, try to use this naming style

---

`.search-input--mini {` changes have been applied